mockgcp/mockbigtable/testdata/logicalview/crud/script.yaml

```diff
@@ -0,0 +1,14 @@
+# Test for `gcloud bigtable logical-views`
+- exec: gcloud bigtable instances create test-instance-${uniqueId} --display-name=test-instance --cluster-config=id=test-cluster-${uniqueId},zone=us-central1-a
```

Comment:
Oh nice! Thanks for adding this. Could you run `WRITE_GOLDEN_OUTPUT=1 E2E_GCP_TARGET=mock go test ./mockgcp/mockgcptests -timeout 60m -run TestScripts/mockbigtable/testdata/logicalview/crud ` to update the test log for this script?